strawberryfields/backends/__init__.py

```diff
@@ -30,7 +30,7 @@
 
     FockBackend
     GaussianBackend
-    ~tfbackend.TFBackend
+    TFBackend
```

Comment:
Going from
![image](https://user-images.githubusercontent.com/24476053/83897422-9690fe00-a723-11ea-9205-ae355195d1b4.png)
to
![image](https://user-images.githubusercontent.com/24476053/83897369-85e08800-a723-11ea-88e5-1dff1484b22d.png)